test/utils_test.cpp

```diff
@@ -23,4 +23,13 @@ TEST(UtilsTestsGroup, trim_test)
     std::string original = " original s ";
     trim(original);
     CHECK_EQUAL("original s", original);
-}
\ No newline at end of file
+}
+
+TEST(UtilsTestsGroup, is_like_numerical_data_test)

```

Comment:
Thanks for writing tests. 
Could you also add one for stream_line to make sure all the delimiters do get parsed properly?